atomic_dbus.py

```diff
@@ -240,22 +244,37 @@ def ImagesList(self):
         args = self.Args()
         args.all=True
         images.set_args(args)
-        i = images.display_all_image_info()
+        i = images.images()
         return json.dumps(i)
 
     # atomic containers section
     # The ImagesDelete method will delete one or more images on the system.
     @slip.dbus.polkit.require_auth("org.atomic.readwrite")
-    @dbus.service.method("org.atomic", in_signature='asbb', out_signature='')
-    def ImagesDelete(self, images, force, remote):
+    @dbus.service.method("org.atomic", in_signature='asbbs', out_signature='i')
+    def ImagesDelete(self, images, force, remote, storage='docker'):
         i = Delete()
         args = self.Args()
         args.delete_targets = images
         args.remote = remote
         args.force = force
+        args.storage = storage
+        args.assumeyes = True
         i.set_args(args)
         return i.delete_image()
 
+    # atomic containers section
+    # The ImagesTag method will create a tag from an existing image.
+    @slip.dbus.polkit.require_auth("org.atomic.readwrite")
+    @dbus.service.method("org.atomic", in_signature='sss', out_signature='i')
+    def ImagesTag(self, src, target, storage='docker'):
```

Comment:
Should not have default here.  Default would be on other side.  dbus is going to force the client to pass three fields.